src/promptflow/promptflow/executor/_errors.py

```diff
@@ -137,10 +133,6 @@ class NodeReferenceError(UserErrorException):
     """Exception raised when node reference not found or unsupported"""
 
     def __init__(self, message="", message_format="", target=ErrorTarget.FLOW_EXECUTOR, **kwargs):
-        if message:
-            message = f"Invalid node reference: {message}"
-        elif message_format:
-            message_format = f"Invalid node reference: {message_format}"
```

Comment:
Do we need to remove `message` and `message_format` from the parameters too?

Reply:
Remove `__init__()` .